test/integration/rise-storage.html

```diff
@@ -535,64 +535,91 @@
+          file.go();
+          server.respond();
+
+          assert(responseHandler.notCalled);
+          done();
+        });
+
+        test("should not fire rise-storage-file-throttled if folder file is not throttled", function(done) {
+          responseHandler = sinon.spy();
+
+          fileFolder.addEventListener("rise-storage-file-throttled", responseHandler);
+          server.respondWith([200, header, JSON.stringify(folderImage)]);
+          fileFolder.go();
+          server.respond();
+
+          assert(responseHandler.notCalled);
+          done();
+        });
+
+        test("should fire rise-storage-file-throttled if bucket file is throttled", function(done) {
           responseHandler = function(response) {
-            assert.isObject(response.detail, "response.detail is an object");
-            assert.isObject(response.detail.error, "response.detail.error is an object");
+            assert.isString(response.detail, "response.detail is a string");
+            assert.equal(response.detail, bucketImage.files[0].selfLink + "?alt=media");
 
             file.removeEventListener("rise-storage-file-throttled", responseHandler);
 
             done();
           };
 
-          // This is required in order to force iron-ajax.html to proceed with firing an error.
-          file.$.storage.verbose = false;
+          bucketImage.files[0].isThrottled = true;
+
           file.addEventListener("rise-storage-file-throttled", responseHandler);
-          server.respondWith([403, header, JSON.stringify(bucketImage)]);
+          server.respondWith([200, header, JSON.stringify(bucketImage)]);
           file.go();
           server.respond();
         });
 
-        test("should fire rise-storage-file-throttled if Storage returns 403 for a folder file", function(done) {
+        test("should fire rise-storage-file-throttled if folder file is throttled", function(done) {
           responseHandler = function(response) {
-            assert.isObject(response.detail, "response.detail is an object");
-            assert.isObject(response.detail.error, "response.detail.error is an object");
+            assert.isString(response.detail, "response.detail is a string");
+            assert.equal(response.detail, folderImage.files[0].selfLink + "?alt=media");
 
             fileFolder.removeEventListener("rise-storage-file-throttled", responseHandler);
 
             done();
           };
 
-          // This is required in order to force iron-ajax.html to proceed with firing an error.
-          fileFolder.$.storage.verbose = false;
+          folderImage.files[0].isThrottled = true;
+
           fileFolder.addEventListener("rise-storage-file-throttled", responseHandler);
-          server.respondWith([403, header, JSON.stringify(folderImage)]);
+          server.respondWith([200, header, JSON.stringify(folderImage)]);
           fileFolder.go();
           server.respond();
```

Comment:
@rodrigopavezi If `responseHandler` isn't called then your test will fail because `done()` will never be called either.